qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -93,6 +132,270 @@ def __init__(self, control_config: Dict[Tuple[int, ...], List[ControlChannel]] =
+        """
+        inst_map = InstructionScheduleMap()
+
+        self.update_inst_map(group=group, cutoff_date=cutoff_date, inst_map=inst_map)
+
+        return inst_map
+
+    def update_inst_map(
+        self,
+        schedules: Optional[set] = None,
+        qubits: Optional[Tuple[int]] = None,
+        group: Optional[str] = "default",
+        cutoff_date: datetime = None,
+        inst_map: Optional[InstructionScheduleMap] = None,
+    ):
+        """Push all schedules from the Calibrations to the inst map.
+
+        This will create instructions with the same name as the schedules.
+
+        Args:
+            schedules: The name of the schedules to update. If None is given then
+                all schedules will be pushed to instructions.
+            qubits: The qubits for which to update the instruction schedule map.
+                If qubits is None then all possible schedules defined by the coupling
+                map will be updated.
+            group: The calibration group from which to draw the parameters. If not specified
+                this defaults to the 'default' group.
+            cutoff_date: Retrieve the most recent parameter up until the cutoff date. Parameters
+                generated after the cutoff date will be ignored. If the cutoff_date is None then
+                all parameters are considered. This allows users to discard more recent values that
+                may be erroneous.
+            inst_map: The instruction schedule map to update. If None is given then the default
+                instruction schedule map (i.e. self._inst_map) will be updated.
+        """
+        inst_map = inst_map or self._inst_map
+
+        for key in self._schedules:
+            sched_name = key.schedule
+
+            if schedules is not None and sched_name not in schedules:
+                continue
+
+            if qubits is not None:
+                inst_map.add(
+                    instruction=sched_name,
+                    qubits=qubits,
+                    schedule=self.get_schedule(
+                        sched_name, qubits, group=group, cutoff_date=cutoff_date
+                    ),
+                )
+
+            else:
+                for qubits_ in self.operated_qubits[self._schedules_qubits[key]]:
+                    try:
+                        inst_map.add(
+                            instruction=sched_name,
+                            qubits=qubits_,
+                            schedule=self.get_schedule(
+                                sched_name, qubits_, group=group, cutoff_date=cutoff_date
+                            ),
+                        )
+                    except CalibrationError:
+                        # get_schedule may raise an error if not all parameters have values or
+                        # default values. In this case we ignore and continue updating inst_map.
+                        pass
+
+    def inst_map_add(
```

Comment:
```suggestion
    def calibrate_parameter(
```
I think something like this is more closer to how we want to use this method.

Reply:
I think the name `calibrate_parameter` is confusing. This implies to perform the act of calibration which could be interpreted as run an experiment. I will leave this as is for now.

Reply:
I think this naming is also confusing because this name looks like alias of `InstructionScheduleMap.add` but this method does't have parameter value replacement. Because this method assumes very specific use case, this should be moved to the helper function of calibration experiments, or renamed to represent that use case.